src/main/java/io/github/robwin/swagger/test/DocumentationDrivenValidator.java

```diff
@@ -109,11 +109,31 @@ private void validatePath(String pathName, Path actualPath, Path expectedPath) {
     }
 
     private void validateDefinition(String definitionName, Model actualDefinition, Model expectedDefinition) {
-        if (expectedDefinition != null) {
+        if (expectedDefinition != null && actualDefinition != null) {
```

Comment:
@RobWin any reason why you didn't have it here before? Had to add it to prevent NPE from happening when doing getRequired()